chart/kubeapps/templates/tiller-proxy-secret.yaml

```diff
@@ -1,4 +1,8 @@
+# The tls ca certificate is only required when tls.verify is set to true, we fail otherwise.
 {{-  if .Values.tillerProxy.tls -}}
+{{- if and (.Values.tillerProxy.tls.verify) (not (.Values.tillerProxy.tls.ca)) -}}
+{{ fail "tillerProxy.tls.ca: A valid CA certificate needs to be provided if tls-verify is set to true." }}
```

Comment:
Can we use the required function, it might make this one-line: https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#know-your-template-functions

Reply:
Good point, I was not aware of it. Will change it